content/docs/command-reference/repro.md

```diff
@@ -97,17 +99,40 @@ up-to-date and only execute the final stage.
 
 ## Options
 
-- `-f`, `--force` - reproduce a pipeline, regenerating its results, even if no
-  changes were found. This executes all of the stages by default, but it can be
-  limited with the `targets` argument, or the `-s`, `-p` options.
+- `targets` (optional argument) - specifies one or more `dvc.yaml` files or
+  specific stage name(s). `./dvc.yaml` by default. E.g.
+  `dvc repro pipes/linear/dvc.yaml`
+
+  Stage names must be defined in `./dvc.yaml`. E.g. `dvc repro train-vision`.
+  Stages in other `dvc.yaml` files can be given using by using a colon `:`
+  following the path to that file. E.g. `models/dvc.yaml:prepare`
+
+  Different things can be provided as targets depending on the flags used (more
+  details in each option), namely:
+
+  - With `-R` you can provide directory paths to search for `dvc.yaml` files in,
+    recursively.
+  - With `--glob`, you can use special patterns (using wildcards) to match
+    groups of stage names.
+
+- `-R`, `--recursive` - looks for `dvc.yaml` files to reproduce in any
+  directories given as `targets`, and in their subdirectories. If there are no
+  directories among the targets, this option has no effect.
+
+- `--glob` - causes the `targets` to be interpreted as wildcard
```

Comment:
Added -R and --glob to `targets` but also moved those options up right next to targets, so I still think it seems a bit redundant.

Reply:
Suggestion:

Keep only

```
Different things can be provided as targets depending on the flags used (more
  details in each option), namely:
```

then a list of examples:

- `dvc reporo -R pipeline` - will look recursively into ....
- `dvc reporo --glob ...` - will execute stages that match pattern
- `dvc repro` - will find dvc.yaml(s) and execute all stages

it's way simpler to grasp the idea, no need to read long long text (if some details are need let's try very hard to squeeze them into each item above)

Reply:
`reporo`? 😆 Sounds funny

OK good idea, applied! PTAL.

Note that issue #1614 is pretty evident this way, should we prioritize it?

![image](https://user-images.githubusercontent.com/1477535/102678126-06208500-416c-11eb-958f-16d6c9ca6e83.png)